apps/snitch_api/lib/snitch_api_web/router.ex

```diff
@@ -10,5 +10,7 @@ defmodule SnitchApiWeb.Router do
     pipe_through(:api)
 
     resources("/orders", OrderController, only: [:index])
+    resources("/taxonomies", TaxonomyController, only: [:index, :show])
+    resources("/taxons", TaxonController, only: [:index, :show])
```

Comment:
I think `taxons` resource must be nested in `taxonomies`. What do you say @SagarKarwande ?

Reply:
Confirmed this from @pkrawat1. Right now it is ok

Reply:
Taxon will be aggregated as per the hierarchy on frontend.

Reply:
Looks fine